sklearn/metrics/tests/test_classification.py

```diff
@@ -1384,6 +1383,24 @@ def test_log_loss():
     loss = log_loss(y_true, y_pred)
     assert_almost_equal(loss, 1.0383217, decimal=6)
 
+    # test labels option
+
+    y_true = [2, 2]
```

Comment:
if you want it as a separate test, then i'd be fine with moving it. no strong opinions either way here.